deploy_tools/cli.py

```diff
@@ -534,6 +583,12 @@ def parse_arg_to_matching_type(arg, type: str):
         if arg.lower() == "false":
             return False
         raise ValueError(f"Expected true or false, but got {arg}")
+    if type.find("address[]") != -1:
```

Comment:
For me it is not clear how you think the addresses should be provided?
I guess `[0x123..., 0x234...]` you want to allow, but what about `0x123, 0x234`? This would work right now but should it? And if it should work, why do we even support it with `[]`?

Reply:
`[0x123...,0x234...]` would work because that is what I tried when I wanted to provide a list of address to the tool, not knowing it wouldn't work.
I made it accept also `0x123,0x234` because that is the second thing I tried when I saw that the first one did not work.

Note that if you put a space `0x1234, 0x1234` it will not work as it will take the first argument `0x1234,` as an address and will complain that it is not.

It is not perfect, but looks better than not accepting arrays at all

Reply:
I would like if there is only one way. So if there is no reason for the `[]` for some reason, please remove the support. 
Just `0x123,0x234` is fine

Reply:
Right now it is broken, because `0x12[3]]],0[x23]]]4` would also work for whatever reason.

Reply:
And we should then document what can be provided I guess. Not necessarily in this PR but please create an issue.